bedrock/newsletter/views.py

```diff
@@ -40,10 +40,6 @@
 
 UNSUB_UNSUBSCRIBED_ALL = 1
 
-# A UUID looks like: f81d4fae-7dec-11d0-a765-00a0c91e6bf6
-# Here's a regex to match a UUID:
-UUID_REGEX = re.compile(r"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$", re.IGNORECASE)
```

Comment:
**question:** I think we still use this same regex in our front end code, but I'm guessing things should still be consistent with this change, right? https://github.com/mozilla/bedrock/blob/main/media/js/newsletter/form-utils.es6.js#L9-L10

Reply:
Yeah Django uses the same regex behind the scenes:
https://github.com/django/django/blob/main/django/urls/converters.py#L29

```python
    regex = "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
```